servers/cromwell/jobs/test/test_jobs_controller.py

```diff
@@ -89,8 +89,7 @@ def test_update_job_labels_returns_200(self, mock_request):
             'test.analysis.outputs': 'gs://project-bucket/test/outputs.txt'
         }
         labels = {}
-        std_err = '/cromwell/cromwell-executions/id/call-analysis/stderr'
-        std_out = '/cromwell/cromwell-executions/id/call-analysis/stdout'
+        backend_log = '/cromwell/cromwell-executions/id/call-analysis/call-analysis-log'
```

Comment:
I'm surprised that `backend_log` has to be explicitly added in here since (presumably) it was already being populated before (hence why we no longer need the `stdout` and `stderr` fields)?